examples/nodejs-nestjs/src/config/arcjet.ts

```diff
@@ -0,0 +1,7 @@
+import { detectBot } from '@arcjet/node';
+
+export const ARCJET_DEFAULT_RULES = [
+    detectBot({
+        mode: "LIVE",
+    }),
```

Comment:
I think we should stick with `shield` being the default rule and all others people add.

Reply:
Agreed on this. Let's swap it for `shield` to protect every route and get them to test the rate limit endpoint as part of the flow with a low limit.